images/debian/Dockerfile

```diff
@@ -0,0 +1,35 @@
+ARG RELEASE
+
+FROM debian:${RELEASE}
+
+# udev is needed for booting a "real" VM, setting up the ttyS0 console properly
+# kmod is needed for modprobing modules
+# systemd is needed for running as PID 1 as /sbin/init
+# Also, other utilities are installed
+RUN apt-get update && apt-get install -y \
+      curl \
+      dbus \
+      kmod \
+      iproute2 \
+      iputils-ping \
+      net-tools \
+      openssh-server \
+      sudo \
+      systemd \
+      udev \
+      vim-tiny \
+      wget && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* && \
+# cleanup machine-id
+    echo "" > /etc/machine-id && echo "" > /var/lib/dbus/machine-id && \
+# inform systemd it is running containerized
+    mkdir -p /run/systemd && echo docker > /run/systemd/container && \
```

Comment:
why do this? it's not running containerized, it's booted as any other VM with its dedicated linux kernel